src/PeopleSDKAdapter.test.js

```diff
@@ -28,14 +32,39 @@ describe('People SDK Adapter', () => {
         done();
       });
     });
+
+    test('emits a Person object with null status on presence plug-in error', (done) => {
+      const errorMessage = 'Presence not enabled for user';
+
+      // SDK presence plug-in fails to return a status
+      mockSDK.internal.presence.get = jest.fn(() => Promise.reject(new Error(errorMessage)));
+
+      peopleSDKAdapter.getMe().subscribe((person) => {
+        expect(person).toMatchObject({
+          status: null,
+        });
+        done();
+      });
+    });
+
+    test('completes after one emission', (done) => {
```

Comment:
Why does `getMe` complete? Wouldn't we want to get presence status updates for "me" as well?

Reply:
this was the design that we agreed on while ago, the goal was to fetch the data one time. if the use needs to listen to the status change, they can invoke `getPerson(myID)` now.

Reply:
What @akoushke said 😛